tests/compose/test-script.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+containers=(
+	webmail
+	imap
+	smtp
+	antispam
+	admin
+	redis
+	antivirus
+	webdav
+#	fetchmail
+	front
+)
+
+# Verbose sleep, to prevent Travis to cancel the build
```

Comment:
Travis builds are only canceled after 10 minutes of log-silence¹, having a function that logs (at the current config) once _before_ sleeping for one minute seems unneccessary to me… am I missing something?

¹ from https://docs.travis-ci.com/user/common-build-problems/#my-builds-are-timing-out :
> Our builds have a global timeout and a timeout that’s based on the output. If no output is received from a build for 10 minutes, it’s assumed to have stalled for unknown reasons and is subsequently killed.

Reply:
At this moment the sleep time is only 1 minute, so it is easier to test the test script. Otherwise the wait would be too long. When we are are happy about the results, the sleep times will need to be increased to cover all sorts of cases:
1. ClamAV to be finished with pulling and updates the virus database
2. In case of Swarm, `docker stack deploy` returns immediately, but it can take significant time for the scheduler to pull in all the images and start containers.
3. Some crashes appear only after a few minutes after start. (ClamAV is doing this when it cannot download the virus database)

So this value can easily extent 10 minutes in the future.

`travis_wait sleep ..m` indeed is a cleaner solution. I will implement this.

Reply:
So the implemation is done with one note. We also use the script for local testing. This means I had to put a fallback sleep for the cases the script is not run on travis.